### PR TITLE
Windows Getting Started POC

### LINKED ISSUE #4941: Provide guidance on alternative to  ‘Android Studio Setup Wizard’ on 'Windows install' page
Page URL: https://docs.flutter.dev/get-started/install/windows
Page source: https://github.com/flutter/website/tree/master/src/docs/get-started/install/windows.md

Description of issue: This page suggests there are two steps to installing Android Studio to provide Android platform dependencies: 1) download and install Studio and then 2) "Start Android Studio, and go through the ‘Android Studio Setup Wizard’"

I've downloaded and installed Android Studio 4.1, and IIRC 3.3. before it, and couldn't figure out how to get to anything called "Setup Wizard" as a discrete step. I didn't see it as an option on the "Welcome to Android Studio" screen, for example.  Can this Windows install page be more specific as to where to go? It seems just installing Android Studio is not enough, as `flutter doctor` still points me back to the Windows install page:

image

### LINKED ISSUE #5735: Explain how to prevent the Android License acceptance error on 'Getting Started' pages
Page URL: https://docs.flutter.dev/get-started/install/linux
Page source: https://github.com/flutter/website/tree/master/src/desktop.md

Description of issue:

Missing a required dependency of the Android SDK Command-line tools. See: https://stackoverflow.com/questions/64708446/flutter-doctor-android-licenses-exception-in-thread-main-java-lang-noclass

### LINKED ISSUE #6913: Add instructions for Android SDK command line tools on 'macOS install' page
### Page URL

https://docs.flutter.dev/get-started/install/macos/

### Page source

https://github.com/flutter/website/tree/main/src/get-started/install/macos.md

### Describe the problem

Android Setup requires adding the sdk's cmdline-tools to your path as mentioned in this StackOverflow question, but this is not mentioned in the doc page.

https://stackoverflow.com/questions/68236007/i-am-getting-error-cmdline-tools-component-is-missing-after-installing-flutter

### Expected fix

Include instructions for adding required tools to your path.

### Additional context

_No response_

### LINKED ISSUE #6934: Change "Set up an editor" to recommend "latest stable" version
### Page URL

https://docs.flutter.dev/get-started/editor/

### Page source

https://github.com/flutter/website/tree/main/src/get-started/editor.md

### Describe the problem

As recommended by @sfshaza2 in #6860 
The "Set up an editor" page currently lists specific versions of editors which can become outdated if the page is left untouched for too long.

### Expected fix

Update the page to recommend the "latest stable" version of Android Studio / VSCode rather than a specific version which could become outdated.
We can use this issue to discuss how this might be implemented.

### Additional context

_No response_

### LINKED ISSUE #7778: Assist users during the download/install process
The recent user retention study showed that completing the multi-step installation process greatly improves user retention after a week, but only a small proportion of the new users complete the process. 

It suggests that we need to better support users during their download/install process, so that they can complete the process without much difficulty.

Some of the suggested actions are:
- Load a webpage about getting started, tips and tricks, some common reasons why the install might fail, etc., while the project is installing
- Offer real-time help for installation
- Showcase a cool, experimental project that can show a few initial things they can try on Flutter (potentially via an online editor that supports hot reload)

cc: @timsneath, @csells

### LINKED ISSUE #8099: Move note about download from GitHub on Windows installation page
### Page URL

https://docs.flutter.dev/get-started/install/windows

### Page source

https://github.com/flutter/website/blob/main/src/get-started/install/_get-sdk-win.md

### Describe the problem

I find putting the note about the Flutter SDK repo before the steps creates a better flow. Also, for people that read more line-by-line, they won't download the zip file before realizing there's another way.

### Expected fix

Created the modification on my fork, here's the [commit](https://github.com/flutter/website/commit/3dd682d0626e661a52274240bcc56d1698f03b4b).

### Additional context

_No response_

### LINKED ISSUE #9333: Refactor Windows Getting Started as PoC of larger update
Refactor the Windows Getting Started page to streamline reader onboarding.

### Windows page 
- [ ] [get-started/install/windows]

#### Desktop (Native) card / page
(`<span class="material-icons-outlined">desktop_windows</span>`)

Includes:
- System Requirements
  - Generic [`_install/reqs/windows.md`]
  - Plus VS Code [`_install/reqs/windows/vscode.md`]
  - Plus VS [`_install/reqs/windows/vs.md`]
- Pre-Requisites [could do this as OS checked]
  - Install VS [`_install/ide/vs.md`]
  - Install VS Code [`_install/ide/vscode/windows.md`]
- Download and extract the Flutter SDK [`_install/flutter/sdk.md`]
  - Set Permanent Path [`_install/path/windows.md`]
  - Run Flutter Doctor [`_run-flutter-doctor-win`]
    - Explain what errors can return
    - Resolve Errors
- Add other targets
  - Web [`_install/add-on/web.md`]
  - Mobile [`_install/add-on/android/windows.md`]

#### Mobile (Android) card / page
(`<span class="material-icons-outlined">phone_android</span>`)

Includes:
- System Requirements
  - Generic [`_install/reqs/windows.md`]
  - Plus VS Code [`_install/reqs/windows/vscode.md`]
  - Plus AS [`_install/reqs/windows/android-studio.md`]
- Pre-Requisites [could do this as OS checked]
  - Install AS [`_install/ide/android-studio/windows.md`]
  - Install VS Code [`_install/ide/vscode/windows.md`]
- Download and extract the Flutter SDK [`_install/flutter/sdk.md`]
  - Set Permanent Path [`_install/path/windows.md`]
  - Run Flutter Doctor [`_install/doctor/windows.md`]
    - Explain what errors can return
    - Resolve Errors
- Add other targets
  - Desktop [`_install/add-on/vs.md`]
  - Web [`_install/add-on/web.md`]

#### Web card / page (Easiest to get to app)
(`<span class="material-icons-outlined">open_in_browser</span>`)

Includes:
- System Requirements
  - Generic [`_install/reqs/windows.md`]
  - Plus VS Code [`_install/reqs/windows/vscode.md`]
  - Plus Chrome [`_install/reqs/windows/chrome.md`]
- Pre-Requisites [could do this as OS checked]
  - Install VS Code [`_install/ide/vscode/windows.md`]
  - Install Chrome [`_install/ide/chrome.md`]
- Download and extract the Flutter SDK [`_install/flutter/sdk.md`]
  - Set Permanent Path [`_install/path/windows.md`]
  - Run Flutter Doctor [`_install/doctor/windows.md`]
    - Explain what errors can return
    - Resolve Errors
- Add other targets
  - Desktop [`_install/add-on/vs.md`]
  - Mobile [`_install/add-on/android/windows.md`]

### PR DESCRIPTION
**Pull request by atsansone**
## Staged

https://tony-flutter-site.web.app/get-started/install/windows/

Fixes #9333 
Fixes #5735 
Fixes #8099 
Fixes #4941 
Fixes #7778 
Fixes #6934 
Fixes #6913

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [ ] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/_includes/docs/install/compiler/android.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+matching quotes. You cannot do _both_.
+
+Look for your `JAVA_HOME` path in your appropriate shell resource file
+and change it to:
+
+```conf
+export JAVA_HOME="/Applications/Android Studio.app/Contents/jre/Contents/Home"
+```
+
+Do not include the backslash between `Android` and `Studio`.
```
Comment (domesticmouse) [inline src/_includes/docs/install/compiler/android.md:103]: What does this line reference? I don't see a backslash in that position?

File: src/_includes/docs/install/devices/android-physical.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+   $ flutter devices
+   ```
+
+   {% endif %}
+
+   By default, Flutter uses the version of the Android
+   SDK where your `adb` tool is based.
+   To use a different Android SDK installation path with Flutter,
+   set the `ANDROID_SDK_ROOT` environment variable
+   to that installation directory.
```
Comment (domesticmouse) [inline src/_includes/docs/install/devices/android-physical.md:50]: Add trailing beeline

File: src/_includes/docs/install/reqs/windows/software.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  Flutter requires the full version of Android Studio, the Android SDK,
+  and the Android emulator.
+- The latest version of [Google Chrome][] to debug JavaScript code for web apps.
+
+{% endif -%}
+
+[Android Studio]: https://developer.android.com/studio/install#windows
+[Visual Studio 2022]: https://learn.microsoft.com/visualstudio/install/install-visual-studio?view=vs-2022
+[Google Chrome]: https://www.google.com/chrome/dr/download/
+[Visual Studio _Code_]: https://code.visualstudio.com/
```
Comment (domesticmouse) [inline src/_includes/docs/install/reqs/windows/software.md:36]: Add trailing new line

File: src/get-started/install/windows/mobile.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  path: /get-started/test-drive
+---
+
+{% include docs/install/reqs/windows/base.md target='mobile' os='Windows' -%}
+
+{% include docs/install/compiler/android.md target='mobile' os='Windows' -%}
+
+{% include docs/install/flutter-sdk.md target='mobile' os='Windows' -%}
+
+{% include docs/install/flutter-doctor.md target='mobile' os='Windows' -%}
```
Comment (domesticmouse) [inline src/get-started/install/windows/mobile.md:16]: Add trailing new line

**Review by domesticmouse — APPROVED** [review]

**Comment by atsansone** [comment]
This should not be merged until the macOS updates have been drafted and approved. They will be merged together.

File: src/_includes/docs/install/compiler/android.md (hunk 1 above)
Comment (sfshaza2) [reply src/_includes/docs/install/compiler/android.md:200]: I see it. There's a backslash in line 191 that you are instructed to remove in line 197.

File: src/_includes/docs/install/flutter-doctor.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+1. To verify your installation of all the components,
+   run the following command.
+
+   ```terminal
+   {{prompt2}} flutter doctor
+   ```
+
+As you chose to develop for {{include.target}},
+you do not need _all_ components.
+If you followed this guide, result of your command should resemble:
```
Comment (sfshaza2) [inline src/_includes/docs/install/flutter-doctor.md:40]:
```suggestion
If you followed this guide, the result of your command should resemble:
```

File: src/_includes/docs/install/flutter-doctor.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+```terminal
+{{prompt2}} flutter doctor -v
+```
+
+Check the output for other software you might need to install
+or further tasks to perform.
+
+If you change the configuration of your Flutter SDK or its related components,
+run `flutter doctor` again to verify the install again.
```
Comment (sfshaza2) [inline src/_includes/docs/install/flutter-doctor.md:59]: Don't use "again" twice.

```suggestion
run `flutter doctor` again to verify the install.
```

**Review by sfshaza2 — APPROVED** [review]
I found a couple of grammatical errors, but nothing serious. Once Brett is happy, then LGTM. I'll go ahead and LGTM now.  @domesticmouse

File: src/_sass/components/_admonitions.scss [hunk 7]
```diff
@@ -0,0 +1,7 @@
+@use '../base/variables' as *;
+@use '../vendor/bootstrap';
+
+aside.alert-icon > p {
+  padding-left: 1.25rem;
+  text-indent: -0.75rem;
+}
```
Comment (domesticmouse) [inline src/_sass/components/_admonitions.scss:7]: nit: add NL at EOF

File: src/assets/js/archive.js [hunk 8]
```diff
@@ -205,7 +205,7 @@ function updateReleaseDownloadButton(releases, base_url, os, arch = '') {
     element.textContent = archiveFilename;
   });
 
-  const genericDownloadLink = document.querySelectorAll('.download-latest-link-filename');
+  const genericDownloadLink = document.querySelectorAll(`.download-latest-link-filename`);
```
Comment (domesticmouse) [inline src/assets/js/archive.js:208]: I'm curious why this change in string markers. From memory, backticks in JS are meant for string interpolation, but I don't see any interpolation happening.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: src/assets/js/archive.js [hunk 9]
```diff
@@ -306,4 +306,4 @@ document.addEventListener("DOMContentLoaded", function(_) {
   if (foundLatestLinux) {
     fetchFlutterReleases('linux', updateDownloadLink, updateDownloadLinkFailed);
   }
-});
+});
```
Comment (domesticmouse) [inline src/assets/js/archive.js:309]: nit: add LF at EOF

**Comment by domesticmouse** [comment]
Just nits mainly

**Pull request merged**